e2e-test/helper.py

```diff
@@ -0,0 +1,165 @@
+
+import json
+import uuid
+import random
+import string
+import wds_client
+import requests
+import time
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+workspace_manager_url = ""
+rawls_url = ""
+leo_url = ""
+
+def setup(bee_name):
+# define major service endpoints based on bee name
+    global workspace_manager_url
+    workspace_manager_url = f"https://workspace.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"workspace_manager_url: {workspace_manager_url}")
+    global rawls_url
+    rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"rawls_url: {rawls_url}")
+    global leo_url
+    leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"leo_url: {leo_url}")
+    return workspace_manager_url, rawls_url, leo_url
+
+# CREATE WORKSPACE ACTION
+def create_workspace(cbas, billing_project_name, header):
+    # create a new workspace, need to have attributes or api call doesnt work
+    api_call2 = f"{rawls_url}/api/workspaces";
+    request_body= {
+      "namespace": billing_project_name, # Billing project name
+      "name": f"api-workspace-{''.join(random.choices(string.ascii_lowercase, k=5))}", # workspace name

```

Comment:
consider allowing the caller to pass in a a string which includes some disambiguating substring that will get included in the generated name (eg: 'wds-cloning-test') that way, if multiple tests start using this utility function, it'll be easier to tell where workspaces got created from for troubleshooting purposes